requirements/dev.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+-e git+https://github.com/kytos-ng/kytos.git@feat/openapi_decorator#egg=kytos
```

Comment:
This commit here will be reverted, but only a moment before this PR gets merged. This is needed since https://github.com/kytos-ng/kytos/pull/352 hasn't landed yet, otherwise unit tests wouldn't pass.

Reply:
Reverted on https://github.com/kytos-ng/mef_eline/pull/304/commits/e28856842409097ecf62d5d8534e4778bd870c4f